README.md

```diff
@@ -2115,7 +2138,15 @@ This endpoint accepts a POST request with a `multipart/form-data` encoded body.
 
 * `wdlSource` - *Required* Contains the WDL file to submit for execution.
 * `workflowInputs` - *Optional* JSON file containing the inputs.  A skeleton file can be generated from [wdltool](https://github.com/broadinstitute/wdltool) using the "inputs" subcommand.
+* `workflowInputs_2` - *Optional* JSON file containing the inputs.
+* `workflowInputs_3` - *Optional* JSON file containing the inputs.
+* `workflowInputs_4` - *Optional* JSON file containing the inputs.
+* `workflowInputs_5` - *Optional* JSON file containing the inputs.
```

Comment:
Actually the REST endpoint now accepts an unlimited number of these, but swagger only accepts up to 5